common/caller_test.go

```diff
@@ -0,0 +1,49 @@
+package common
```

Comment:
CallerName is not coming from the runtime, but is simply using some of the runtime functions, so I should be verified that the method results respects the method description